src/transport/RendezvousSession.cpp

```diff
@@ -198,20 +174,10 @@ void RendezvousSession::OnSessionEstablished()
 
     InitPairingSessionHandle();
 
-    // TODO: This check of BLE transport should be removed in the future, after we have network provisioning cluster and ble becomes
-    // a transport.
-    if (mParams.GetPeerAddress().GetTransportType() != Transport::Type::kBle || // For rendezvous initializer
-        mPeerAddress.GetTransportType() != Transport::Type::kBle)               // For rendezvous target
-    {
-        UpdateState(State::kRendezvousComplete);
-        if (!mParams.IsController())
-        {
-            OnRendezvousConnectionClosed();
-        }
-    }
-    else
+    UpdateState(State::kRendezvousComplete);
+    if (!mParams.IsController())
     {
-        UpdateState(State::kNetworkProvisioning);
```

Comment:
I guess we need to remove all the code that was doing something when `RendezvousSession` was in this state ?

Reply:
I found there are lots of code that need to be deleted / modified if we want to remove the network provisioning protocol.
I would expect another PR, say "Remove deprecated NetworkProvisioning protocol"